contracts/src/pre-deploy/ERC1967Proxy.sol

```diff
@@ -0,0 +1 @@
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
```

Comment:
Don't do this, directly import the contract in js code

Reply:
Do this in js:
`import ERC1967Proxy from "@openzeppelin/contracts/build/contracts/ERC1967Proxy.json"; `

See `contracts/deploy/util.ts:4`

Reply:
I changed the genesis generation so now if there is no hre compilation artifact, it looks in `node_modules/@openzeppelin`.

This comes with the limitation that we can't read the storage layout from contracts imported that way, which is fine in the case of the Proxy but something to keep in mind for the future.

Reply:
yeah manually providing the storage slots will still work, just not resolving var names.

how do you think the versioning should work? only think I can think of rn is having the option to provide a contract hash in `base_genesis.json` and then check if that matches during genesis generation?